images/baremetal/Dockerfile.ci

```diff
@@ -0,0 +1,24 @@
+# This Dockerfile is a used by CI to publish kni/origin-v4.0:installer
+# It builds an image containing only the openshift-install.
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
```

Comment:
Were you able to get this to build using go 1.10? I've been worried we've needed to pull in newer deps that have officially dropped 1.10 support.

Reply:
Yep, it builds

```
+ oc --config release-kubeconfig logs -f kni-installer-4.0.0-0.ci-2019-04-17-133604-build

Pulling image registry.svc.ci.openshift.org/openshift/origin-v4.0:base ...
--> FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 as builder
--> WORKDIR /go/src/github.com/openshift-metalkube/kni-installer
--> COPY . .
--> RUN TAGS="libvirt ironic" hack/build.sh
+ minimum_go_version=1.10
++ go version
++ cut -d ' ' -f 3
+ current_go_version=go1.10.8
++ version 1.10.8
++ IFS=.
++ printf '%03d%03d%03d\n' 1 10 8
++ unset IFS
++ version 1.10
++ IFS=.
++ printf '%03d%03d%03d\n' 1 10
++ unset IFS
+ '[' 001010008 -lt 001010000 ']'
+ LAUNCH_PATH=/go/src/github.com/openshift-metalkube/kni-installer
++ dirname hack/build.sh
+ cd hack/..
++ go list -e -f '{{.Dir}}' github.com/openshift-metalkube/kni-installer
+ PACKAGE_PATH=/go/src/github.com/openshift-metalkube/kni-installer
+ test -z /go/src/github.com/openshift-metalkube/kni-installer
+ LOCAL_PATH=/go/src/github.com/openshift-metalkube/kni-installer
+ test /go/src/github.com/openshift-metalkube/kni-installer '!=' /go/src/github.com/openshift-metalkube/kni-installer
+ MODE=release
++ git describe --always --abbrev=40 --dirty
++ git rev-parse --verify 'HEAD^{commit}'
+ LDFLAGS=' -X github.com/openshift-metalkube/kni-installer/pkg/version.Raw=1c49351ca5910af868284ee2cb1b05f4cb89adad-dirty -X github.com/openshift-metalkube/kni-installer/pkg/version.Commit=1c49351ca5910af868284ee2cb1b05f4cb89adad'
+ TAGS='libvirt ironic'
+ OUTPUT=bin/kni-install
+ export CGO_ENABLED=0
+ CGO_ENABLED=0
+ case "${MODE}" in
+ LDFLAGS=' -X github.com/openshift-metalkube/kni-installer/pkg/version.Raw=1c49351ca5910af868284ee2cb1b05f4cb89adad-dirty -X github.com/openshift-metalkube/kni-installer/pkg/version.Commit=1c49351ca5910af868284ee2cb1b05f4cb89adad -s -w'
+ TAGS='libvirt ironic release'
+ test -n ''
+ test '' '!=' y
+ go generate ./data
writing assets_vfsdata.go
+ echo 'libvirt ironic release'
+ grep -q libvirt
+ export CGO_ENABLED=1
+ CGO_ENABLED=1
+ go build -ldflags ' -X github.com/openshift-metalkube/kni-installer/pkg/version.Raw=1c49351ca5910af868284ee2cb1b05f4cb89adad-dirty -X github.com/openshift-metalkube/kni-installer/pkg/version.Commit=1c49351ca5910af868284ee2cb1b05f4cb89adad -s -w' -tags 'libvirt ironic release' -o bin/kni-install ./cmd/kni-install
--> FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base as 1
--> COPY --from=builder /go/src/github.com/openshift-metalkube/kni-installer/bin/kni-install /bin/openshift-install
```